sdk/cmake/baseaddress.cmake

```diff
@@ -196,7 +196,7 @@ set(baseaddress_load_notifications             0x73cd0000)
 set(baseaddress_loadperf                       0x73ca0000)
 set(baseaddress_localmon                       0x73c40000)
 set(baseaddress_localspl                       0x73b50000)
-set(baseaddress_localspl_apitest               0x73b20000)
+set(baseaddress_localspl_apitest.dll           0x73b20000) #module name corrected manually
```

Comment:
Why does this one need the extension?

Reply:
because otherwise it won't get a base address, see https://jira.reactos.org/browse/CORE-14831
I m not sure, why this one is so special, but we touched and renamed that stuff multiple times in the past, and composed part of its name. That was around 0.4.7 to 0.4.9 iirc. Thomas Faber might remember, he was involved in that. Our python script was never able to cover that dll correctly.

Reply:
Ah, this is because the `baseaddress_*` names there are based on the cmake build targets. And the generated `localspl_apitest` dll cmake build target name contains the extension (in order to disentangle it from the other build target that would otherwise have the same name, but builds the test .exe): see the first results there:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=localspl_apitest